src/index.js

```diff
@@ -119,7 +119,8 @@ export const createStyleFunction = async (esriLayerInfoJson, mapProjection) => {
  * @property {Array<import('./types').StyleType>} [styles.featureStyles]
  * @property {Array<import('./types').LabelType>} [styles.labelStyles]
  */
-export const readEsriStyleDefinitions = ({ renderer, labelingInfo }) => {
+export const readEsriStyleDefinitions = async (drawingInfo) => {
+	  const { renderer, labelingInfo, transparency } = drawingInfo;
```

Comment:
I will update this whitespace later. My IDE prefers tabs over spaces and I didnt catch this one. 

Would you be open to a separate pull request adding Prettier for autoformatting?